src/components/common/applicant-info-1.vue

```diff
@@ -390,6 +390,9 @@ import { ActionMixin } from '@/mixins'
 import { ActionBindingIF } from '@/interfaces/store-interfaces'
 import { NameRequestI } from '@/interfaces'
 
+// List Data
+import { CanJurisdictions, IntlJurisdictions } from '@/list-data'
```

Comment:
You should have these in `this` already. See https://github.com/bcgov/namerequest/blob/ae1da6cd7e7aa9dda104ce524fbbc9e7d865c135/src/main.ts#L46.

Reply:
ah, sorry. I thought it was preferential to go without the globally scoped variables. 
Was i thinking backwards here?

Reply:
In the spirit of "starting to do things the better way", your code is the preferred approach (ie, only import what you need). This will be one less place we have to fix when we eventually get rid of the globally scoped variables :)

As you were.